templatestore/frontend/src/components/base64.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
```

Comment:
rename the file to `templatestore/frontend/src/utils.js`

Reply:
ok